app/webpacker/styles/category.scss

```diff
@@ -1,5 +1,4 @@
 .category {
-  background-color: $white;
```

Comment:
Removing this as category probably shouldn't have it's own background colour, that feels like it should be up to the section it's placed in. Currently, placing category on a grey background shows white gaps between cards because of this. It doesn't seem to affect existing category cards but worth double checking!